src/httpserver.cpp

```diff
@@ -14,6 +14,7 @@
 #include <ui_interface.h>
 
 #include <memory>
+#include <set>
```

Comment:
unordered_set?

Reply:
IIRC for small collections std::set is preferable. I can change if that sounds better to you.